backend/tests/routes/test_user_routes.py

```diff
@@ -1,6 +1,25 @@
 from fastapi.testclient import TestClient
 from sqlmodel import Session, select
-from models import User
+from models import User, Client
+
+
+def test_get_phone_number_in_international_format(session: Session) -> None:
+    user = session.exec(select(User).where(User.phone_number == "+999")).first()
+    assert str(user) == "+999"
+    assert user.serialize() == {
+        "id": user.id,
+        "phone_number": "+999",
+        "login_code": None,
+    }
+
+    client = session.exec(
+        select(Client).where(Client.phone_number == "+998")
+    ).first()
+    assert str(client) == "+998"
+    assert client.serialize() == {
+        "id": client.id,
+        "phone_number": "+998",
+    }
```

Comment:
Maybe also add 1) a test setting the phone number, to show the `+` is stripped by the user class and 2) a test setting the phone number to (say) `+123 45` and seeing that throws an error, so that we cannot get bad phone numbers.

Reply:
Nice one!